Live/Classes/Inventory.h

```diff
@@ -0,0 +1,27 @@
+#ifndef __INVENTORY__H__
+#define __INVENTORY__H__
+
+#include <vector>
+#include "Item.h"
+
+#define DEFAULT_MAX_WEIGHT 40
+#define MAX_ITEMS 6
+
+class Inventory {
+private:
+    unsigned int max_weight;
+    unsigned int current_weight;
+    unsigned int num_items;
+    Item *items[MAX_ITEMS];
+public:
+    Inventory() {
```

Comment:
If you are putting any of the functions in a separate cpp file, put the contructor there too.